worktime-tracker-dev/components/account/ResetPassword.tsx

```diff
@@ -0,0 +1,43 @@
+import { useState } from "react";
+import AccountInput from "./Input";
+
+export default function resetPassword() {
+    const [inputValue, setInputValue] = useState<string>("");
```

Comment:
React Hook "useState" is called in function "resetPassword" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter. React Hook names must start with the word "use".

Reply:
revised